src/main/org/apache/tools/ant/taskdefs/optional/junitlauncher/LauncherSupport.java

```diff
@@ -45,14 +45,7 @@
 import org.junit.platform.launcher.listeners.SummaryGeneratingListener;
 import org.junit.platform.launcher.listeners.TestExecutionSummary;
 
-import java.io.File;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
-import java.io.PipedInputStream;
-import java.io.PipedOutputStream;
-import java.io.PrintStream;
-import java.io.UncheckedIOException;
+import java.io.*;
```

Comment:
Can you undo the `*` import and use the previous existing imports plus one new for the BufferedOutputStream?

Reply:
done